modules/plugin/coverage-multidim/netcdf/src/test/java/org/geotools/coverage/io/netcdf/CreateIndexer.java

```diff
@@ -279,27 +281,29 @@ private static void addDomainsToCoverage(String schemaAttribs, StringBuilder bui
 
     private static void getAttributes(Set<String> timeAttributes, Set<String> elevationAttributes,
             Element root) throws JDOMException {
-        List<Element> schemaAttributes = XPath.selectNodes(root,
+        List<?> schemaAttributes = XPath.selectNodes(root,
                 "coverages/coverage/schema/attributes");
-        for (Element e : schemaAttributes) {
-            String attributes = e.getText();
-            String[] attribs = attributes.split(",");
-            for (String attrib : attribs) {
-                if (attrib.contains(TIME_ATTRIB_TYPE)) {
-                    String[] nameTypePair = attrib.split(":");
-                    String name = nameTypePair[0];
-                    if (!timeAttributes.contains(name)) {
-                        timeAttributes.add(name);
-                    }
-
-                } else if (attrib.contains(ELEVATION_ATTRIB_TYPE_FLOAT) || attrib.contains(ELEVATION_ATTRIB_TYPE_DOUBLE)) {
-                    String[] nameTypePair = attrib.split(":");
-                    String name = nameTypePair[0];
-                    if (!elevationAttributes.contains(name)) {
-                        elevationAttributes.add(name);
+        for (Object e : schemaAttributes) {
+        	if (e instanceof Element) {
```

Comment:
Does not follow coding conventions (since this part was fully intented anyways,  you could select it and IDE reformat it)